pkg/controller/metrics.go

```diff
@@ -27,26 +27,26 @@ type metrics struct {
 }
 
 const (
-	ACTION_SET_COUNTER_VEC_LABEL_RES         = "resolution"
-	ACTION_SET_COUNTER_VEC_LABEL_RES_SUCCESS = "success"
-	ACTION_SET_COUNTER_VEC_LABEL_RES_FAILURE = "failure"
+	ACTION_SET_COUNTER_VEC_LABEL_RES         = "resolution" //nolint:stylecheck // Already defined in docs
+	ACTION_SET_COUNTER_VEC_LABEL_RES_SUCCESS = "success"    //nolint:stylecheck // Already defined in docs
+	ACTION_SET_COUNTER_VEC_LABEL_RES_FAILURE = "failure"    //nolint:stylecheck // Already defined in docs
```

Comment:
What is the error we are getting?

Reply:
Could you double-check if the metrics setting is enabled by default? Also, the label values are going to be the same right? We would only be changing the constant used to refer to them

Reply:
checked. Only `ACTION_SET_COUNTER_VEC_LABEL_RES` variable is used in documentation, others are not referenced anywhere but code. Refactored those.